src/librustdoc/html/render/write_shared.rs

```diff
@@ -303,6 +303,10 @@ pub(super) fn write_shared(
         )?;
     }
 
+    if cx.shared.layout.scrape_examples_extension {
+        write_minify("scrape-examples.js", static_files::SCRAPE_EXAMPLES_JS, cx, options)?;
```

Comment:
This needs to use cx.write_minify, ToolchainSpecific is not correct.

Reply:
I think I'm going to get rid of the write_minify closure altogether, it's too easy to get wrong.